dcos/marathon.py

```diff
@@ -556,6 +565,23 @@ def get_task(self, task_id):
 
         return task
 
+    def get_app_schema(self):
+        """Returns app json schema
+
+        :returns: application json schema
+        :rtype: json schema or None if endpoint doesn't exist
+        """
+
+        version = self.get_version()
+        schema_version = LooseVersion("0.9.0")
+        if version < schema_version:
```

Comment:
Just to clarify, did you run the integration test suite against the RC?  If so :shipit:.